src/rpc/blockchain.cpp

```diff
@@ -1449,7 +1454,7 @@ static const CRPCCommand commands[] =
   //  --------------------- ------------------------  -----------------------  ------ --------
     { "blockchain",         "getbestblockhash",       &getbestblockhash,       true,  {} },
     { "blockchain",         "getbestsaplinganchor",   &getbestsaplinganchor,   true,  {} },
-    { "blockchain",         "getblock",               &getblock,               true,  {"blockhash","verbose"} },
+    { "blockchain",         "getblock",               &getblock,               true,  {"blockhash","verbosity"} },
```

Comment:
To be backward compatible this should be `"verbose|verbosity"`
(source: https://github.com/bitcoin/bitcoin/pull/8704/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR1437)

But then `src/rpc/server.cpp` would have to be updated as well to recognise `"|"` as seperator
Like https://github.com/bitcoin/bitcoin/pull/8704/files#diff-019ee7d5e66b74eac42199f64e08cd0e90af4603bb3c105e294665ea4b411219R437-R446